README.md

```diff
@@ -1 +1,45 @@
-# operator-workflows
\ No newline at end of file
+# operator-workflows
+
+## Description
+
+This repository includes several parametrized CI workflows to be used for both kubernetes and machine charms.
+
+## Usage
+
+To use the workflows in your repository, just reuse them including the `secrets: inherit` key. Examples of usage for every workflow can be found at the (indico operator)[https://github.com/canonical/indico-operator/blob/main/.github/workflows].
+
+The following workflows are available:
+
+* test: executes the default tox targets defined in the `tox.ini`file and generates a plain text report. The following parameters are available for this workflow:
```

Comment:
Looks like you're missing a whitespace after `tox.ini`.

Reply:
Indeed. Thx